src/main/resources/config/liquibase/changelog/20210922211100_changelog.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.9.xsd">
+    <changeSet author="schlesinger" id="20210922211100">
+        <sql>
+            DELETE FROM reaction
```

Comment:
please add some natural language comments what this sql query will do

Reply:
and a better formatting would be nice (inner queries should e.g. be indented)

Reply:
done in 3e661c3